src/test/java/com/advanon/pdfsignatures/SignatureMetadataTest.java

```diff
@@ -0,0 +1,56 @@
+package com.advanon.pdfsignatures;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+import org.junit.jupiter.api.Test;
+
+
+class SignatureMetadataTest {
+  @Test
+  public void itSetsAndGetsReason() {
+    SignatureMetadata signaturemetaData = new SignatureMetadata() {
+      {
+        setReason("My reason");
```

Comment:
Let's say this is fine because we are in a test. But it seems clear up to this point we need a Builder for SignatureMetadata